scanners/github-cloned-repo-checks/src/yaml-parser.js

```diff
@@ -0,0 +1,17 @@
+// fs docs node https://nodejs.org/docs/v0.3.4/api/fs.html
+import * as fs from 'fs';
```

Comment:
Similar to the comment on the `README.md` document, I'm wondering if logic for parsing `product.yaml` should live in the "graph updater" component rather than the "endpoint scanner" component.

The idea being that "graph updater" handles stuff related to the graph structure, and "endpoint scanner" handles stuff related to attributes of given endpoints (e.g. accessibility scans).

Reply:
Yes absolutely - this was pre-last week conversation, getting this working.